com.unity.render-pipelines.core/Runtime/Debugging/Prefabs/Scripts/DebugUIHandlerPanel.cs

```diff
@@ -56,13 +57,26 @@ public void SelectPreviousItem()
             Canvas.SelectPreviousPanel();
         }
 
+        /// <summary>
+        /// Scrollbar value clicked via mouse/touch.
+        /// </summary>
+        public void OnScrollbarClicked()
+        {
+            DebugManager.instance.SetScrollTarget(null); // Release scroll target
+        }
+
+        internal void SetScrollTarget(DebugUIHandlerWidget target)
+        {
+            m_ScrollTarget = target;
+        }
+
         // TODO: Jumps around with foldouts and the likes, fix me
```

Comment:
Can we remove this TODO and create a Jira task for this?

Reply:
Yes still valid. I have a jira here: https://jira.unity3d.com/browse/XPIPELINE-129

Reply:
Okay great 👍   
I would personally prefer that we would remove the TODO since the task is tracked elsewhere but this is a minor thing.